src/JSInterop/Microsoft.JSInterop/src/Microsoft.JSInterop.csproj

```diff
@@ -8,11 +8,13 @@
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
     <Nullable>enable</Nullable>
     <Trimmable>true</Trimmable>
+    <DefineConstants>$(DefineConstants);JS_INTEROP</DefineConstants>
```

Comment:
TBH, given it's `internal`, I'm not really sure why we bother going through the trouble of changing the namespace in `ArrayBuilder.cs`. I know you're following on from the existing pattern with ignitor and webview, but I don't really know why they bother doing it either.

This is just to say that if you want to do this, fine, and if you don't want to bother with it, I'd support that too!

Reply:
Thanks, think I'll keep it in that case just to stick with the general pattern elsewhere as you mentioned.